stats/cloud/data.go

```diff
@@ -213,7 +213,7 @@ func (am *AggregatedMetric) Add(t time.Duration) {
 	am.sumD += t
 }
 
-// Calc populates the float fields for min and max and calulates the average value
+// Calc populates the float fields for min and max and caclulates the average value
```

Comment:
calculates ?

Reply:
FFS :man_facepalming:

Reply:
I'm starting to worry that I have dyslexia or something... Though this is probably because I'm trying to kick caffeine... Fixed